slides/syntroduction/slides_examples.adoc

```diff
@@ -0,0 +1,357 @@
+// TODO: Add your personal information
```

Comment:
I guess we do not need the example in here. Also applies to the unused assets.

Reply:
It helps me to write the slides when I can open the example file in the same editor  from the same directory in a second tab and lookup stuff which I forgot.